code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -16,6 +16,7 @@
 	/// Used to handle rotation properly, should only be 1, 4, or 8
 	var/possible_dirs = 4
 	var/colorable = FALSE
+	var/override_layer = FALSE // makes the player sprite always display above the chair (used for benches)
```

Comment:
oh yeah, this should be a triple comment on the line before (see above) for vscode integration

```
/// ...
var/
```